From e32ba39cdbfb150c3a0ca1769a43a0074fb0a951 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 6 Sep 2026 02:42:15 +0800
Subject: [PATCH 6/6] AOSCOS: fix(desktop): do not show CUPS configuration
 shortcut in KDE

KDE comes with its own printer configuration utility.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 desktop/cups.desktop.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/desktop/cups.desktop.in b/desktop/cups.desktop.in
index b97cadbc6..96d7f5099 100644
--- a/desktop/cups.desktop.in
+++ b/desktop/cups.desktop.in
@@ -4,6 +4,7 @@ Exec=@CUPS_HTMLVIEW@ http://localhost:631/
 Icon=cups
 StartupNotify=false
 Terminal=false
+NotShowIn=KDE;
 Type=Application
 Name=Manage Printing
 Comment=CUPS Web Interface
-- 
2.55.0

